src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -102,15 +113,16 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     * make sure we're only listening for messages from the correct source in the
     * registerPostMessageListener method.
     */
-    buildVisualforceOriginUrl(domain) {
+    buildVisualforceOriginUrls(domain) {
         let url = `https://${domain}--c.visualforce.com`;
-
-        const currentNamespace = getNamespace(DATA_IMPORT_PAYMENT_AUTHORIZATION_TOKEN_FIELD.fieldApiName);
+        let alternateUrl = `https://${domain}--c.visual.force.com`;
```

Comment:
I believe we need to include the 'pod' information in here as well (or else modify the checking logic somehow) - the alternate format looks like this:

${domain}--npsp.cs18.visual.force.com

And I believe we need to grab the  "cs18" section from somewhere - it's an additional piece of information that isn't in the shortened format. I think there are examples of grabbing it like so: 

String pod = [SELECT InstanceName FROM Organization].InstanceName.toLowerCase();

Which appears to work!

Reply:
Done!